requirements/base.in

```diff
@@ -5,4 +5,4 @@ Django
 pyjwt[crypto]>=2.1.0 # depends on newer jwt.decode and jwt.encode functionality
```

Comment:
Nit: I meant that this comment would also have spaces to be aligned with the other. :)